packages/components/breadcrumbs/breadcrumbs.scss

```
@@ -0,0 +1,64 @@
+@use '../core/styles/common/tokens' as tokens;
```

```suggestion
@use '../core/styles/common/tokens';
```
можно не писать `as ...`, если только желаешь переименовать